metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/integration/ParserDriver.java

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
How bad would it be to keep both types of tests?  If we actually wanted to do the old integration tests for certain types of build ( outside of travis )?  Like validation of releases or pre-pr commit?